heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/YarnLauncher.java

```diff
@@ -161,7 +162,7 @@ Configuration getHMDriverConf() {
         .set(HeronDriverConfiguration.HTTP_PORT, 0)
         .set(HeronDriverConfiguration.VERBOSE, false)
         .set(YarnDriverConfiguration.QUEUE, queue)
-        .set(DriverConfiguration.DRIVER_MEMORY, driverMemory)
+        .set(DriverConfiguration.DRIVER_MEMORY, driverMemory.asMegabytes())
```

Comment:
I'm confused here. Shouldn't `DRIVER_MEMORY` accept value of type int, according to [reef](https://github.com/apache/reef/blob/master/lang/java/reef-common/src/main/java/org/apache/reef/client/DriverConfiguration.java#L59)?